PocketKit/Sources/PocketKit/MyList/ItemsList/SavedItemsList/SavedItemsListViewModel.swift

```diff
@@ -556,15 +556,6 @@ extension SavedItemsListViewModel {
         handleFilterSelection(with: filter, sender: sender)
 
         fetch()
-
-        var snapshot = buildSnapshot()
```

Comment:
Removing, `fetch()` will implicitly trigger the FetchedResultsController to update with any changes to the user.